Makefile

```diff
@@ -269,3 +270,14 @@ vendor/check: vendor/fix
 vendor/fix:
 	go mod tidy
 	go mod vendor
+
+.PHONY: manifest/check/image_mirror_mapping
+manifest/check/image_mirror_mapping:
+ifneq ( ,$(findstring image_mirror_mapping,$(IMAGE_MAPPINGS)))
+		$(error found image_mirror_mapping in $(IMAGE_MAPPINGS))
+else
+		@echo "No image_mirror_mapping files found in manifests directory"
+endif
+
+.PHONY: manifest/check
+manifest/check: manifest/check/image_mirror_mapping
```

Comment:
I was thinking it might be good to keep all related make targets in separate make files see https://github.com/integr8ly/integreatly-operator/tree/master/make. Should we consider moving this to a manifest.mk file?

This will also allow us to add an OWNERS file in there so that we can add a few more people that can approve these changes.

Reply:
Sure sounds good to me to keep things separated. I've pushed the change